lib/engines/content_block_manager/app/models/content_block_manager/content_block/schema/embedded_schema.rb

```diff
@@ -40,6 +40,20 @@ def permitted_params
           end
         end
 
+        def govspeak_enabled?(field:, nested_object_key: nil)
+          return top_level_govspeak_enabled_fields.include?(field) unless nested_object_key
+
+          govspeak_enabled_fields_for_nested_object(nested_object_key).include?(field)
+        end
+
+        def govspeak_enabled_fields_for_nested_object(object_key)
+          body.dig("properties", object_key, "govspeak_enabled") || []
+        end
+
+        def top_level_govspeak_enabled_fields
+          body.dig("properties", "govspeak_enabled") || []
```

Comment:
As above, we'll need to make sure we rename this

Reply:
Might be worth having a `GOVSPEAK_ENABLED_PROPERTY_KEY` constant so we don't have to reference it every time

Reply:
done, thks